adding a simple map without markers

haithhawk opened the pull request.

haithhawk committed:
adding a simple map without markers

idantb101 reviewed (approved):
Looked at this, looks nice.

sophiavocado commented:
so what happened to the api key? it just works without it? 🤔

NickP123 reviewed (approved):
Can we amend the class name so that it is easier to reference your fancyComponent going fwd within the bigger project : 0 ) . Thanks!

haithhawk committed:
changed the component and class name
and adding instructions about how to run the servers in the readme relates to issue #24 and #25
